src/resources/Microsoft.Authorization/policySetDefinitions/Enforce-ACSB.json

```diff
@@ -0,0 +1,88 @@
+{
+    "name": "Enforce-ACSB",
+    "type": "Microsoft.Authorization/policySetDefinitions",
+    "apiVersion": "2021-06-01",
+    "scope": null,
+    "properties": {
+        "policyType": "Custom",
+        "displayName": "Enforce Azure Compute Security Benchmark compliance auditing",
+        "description": "Enforce Azure Compute Security Benchmark compliance auditing for Windows and Linux virtual machines.",
+        "metadata": {
+            "version": "2.0.0",
+            "category": "Guest Configuration",
+            "source": "https://github.com/Azure/Enterprise-Scale/",
+            "alzCloudEnvironments": [
+                "AzureCloud"
+            ]
+        },
+        "parameters": {
+            "includeArcMachines": {
+                "type": "String",
```

Comment:
is this really a string and not a `bool`?

Reply:
It is, I checked, wow!

Can you therefore included an `allowedValues` array with 

```json
[
"true",
"false"
]
```